mono/mini/mini.c

```diff
@@ -4402,13 +4402,26 @@ mono_add_patch_info (MonoCompile *cfg, int ip, MonoJumpInfoType type, gconstpoin
 	g_assert_not_reached ();
 }
 
+#else // DISABLE_JIT
+
+guint8*
+mini_realloc_code_slow (MonoCompile *cfg, int size)
+{
+	const int EXTRA_CODE_SPACE = 16;
```

Comment:
Couldn't this const be shared with the one on mini.h?

Reply:
As a #define (or const static) yes. I was limiting the scope but agree it isn't great.